---
title: MOLECULAR CHARACTERISATION OF CARBAPENEM-RESISTANT Klebsiella pneumoniae ISOLATED FROM NEONATES AND ADULTS WITH BLOODSTREAM INFECTIONS ADMITTED AT MUHIMBILI NATIONAL HOSPITAL BETWEEN JULY 2021 AND MARCH 2022.
author:
  - name: Lawrence Amon Mapunda
    email: amonlawrence65@gmail.com
    affiliation: Tanzania National Public Health Laboratory
    correspondingauthor: true
    footnote: 1
  - name: Henerico Shimba
    email: henricus20@ymail.com
    affiliation: Bugando Medical Centre
  - name: Nsiande Lema
    email: nsiandelema123@gmail.com 
    affiliation: Tanzania Field Epidemiology and Laboratory Training Program
    footnote: 2
  - name: Joel Manyahi
    email: manyahijoel@yahoo.com 
    affiliation: Muhimbili University of Health and Allied Sciences
    footnote: 2
address:
  - code: Tanzania National Public Health Laboratory
    organization: Tanzania National Public Health Laboratory
    addressline: Barack Obama Drive
    city: Dar es Salaam
    state: Ilala
    postcode: 0000
    country: Tanzania
  - code: Another University
    organization: Department
    addressline: A street 29
    postcode: 2054 NX
    city: Manchester,
    country: The Netherlands
footnote:
  - code: 1
    text: "This is the first author footnote."
  - code: 2
    text: "Another author footnote."
abstract: |
  This is the abstract.
  It consists of two paragraphs.
keywords: 
  - carbapenem resistance
  - Klebsiella pneumoniae
  - Whole genome sequencing
  - Klebsiella variicola
journal: "Journal of Global Antimicrobial Resistance"
date: "`r Sys.Date()`"
linenumbers: false
numbersections: true
bibliography: mybibfile.bib
biblio-style: elsarticle-harv # author year style for natbib - use 'elsarticle-num' or 'elsarticle-num-names' for numbered scheme
classoption: preprint, 3p, authoryear # remove authoryear is not using `elsarticle-harv`
# Use a CSL with `citation_package = "default"`
# csl: https://www.zotero.org/styles/elsevier-harvard
output: 
  rticles::elsevier_article:
    keep_tex: true
    citation_package: natbib
---
# Introduction

# Material and methods

## Study setting and design
The study used clinical isolates from Muhimbili National Hospital (MNH); MNH is National Referral Hospital and University Teaching Hospital with a 1,500-bed facility, attending 1,000 to 1,200 outpatients per week and admitting 1,000 to 1,200 inpatients per week. (67). In a recent study at MNH, the prevalence of bloodstream infections was 11.4%, with a case fatality of 37% (28). At MNH, the detection of bloodstream infections is done through microbiological culture techniques and the use of the Bactec® machine(Biomeriux, France), which is used for blood cultures. The study was a retrospective descriptive study.
 
## Study population, sample size and selection
The study included carbapenem-resistant *Klebsiella pneumoniae* isolates from patients with bloodstream infections at MNH between July 2021 and March 2022. The isolate was termed CRKP if it was resistant to meropenem and/or imipenem by disc diffusion method. Conveniently 8 samples were selected based on their ascending zone of inhibition.
 
## Bacteria isolate culture and identification.
A total of 16 isolates of carbapenem-resistant *Klebsiella pneumoniae* from stored isolates were sub-cultured at the NPHL microbiology laboratory. *Klebsiella pneumoniae* was isolated by using conventional methods on MacConkey agar using the standard microbiological technique. Isolates from the stored vial were subcultured on MacConkey agar (Oxoid, UK) using a 10µl disposable loop into four quadrants. The agar plates were then incubated at 37°c overnight in an incubator. Identification of isolates for confirmation was done using the MALDI-TOF system VITEK MS machine (Biomerieux, France). A fresh isolate (24 hours) of E. coli ATCC 8739 was spotted on each Vitek MS slide acquisition group as a control organism. In addition, thin layers of fresh colonies of the CRKPs were spotted on the slides using the spotting pen (pick me pen). Later, each spot, 1 µl of the matrix was added and was left to air dry. The slide was later loaded on the machine (VITEK MS)for the identification of isolates.
 
## Antimicrobial sensitivity testing

Antimicrobial susceptibility testing Susceptibility test was done using  Muller-Hinton agar a disk diffusion methods (Kirby-Bauer) for Meropenem, Imipenem, ampicillin, amikacin, tetracycline, Sulfamethoxazole-Trimethoprim, ceftazidime, ceftriaxone, gentamicin and ciprofloxacin. The results were interpreted according to Clinical and Laboratory Standards Institute (CLSI-M100 32nd edition) guidelines (69).  3-5 colonies were picked using a sterile loop and emulsified in 4-5 ml sterile saline to make a bacteria suspension. The suspension was adjusted to 0.5 McFarland standard with sterile saline solution. A sterile cotton swab was dipped in the suspension and streaked on the Muller-Hinton agar's entire surface. Five discs were placed per plate, and the plates were incubated at 37°c in the incubator for 18 hours.

## DNA extraction and quantification

DNA from the selected carbapenem-resistant Klebsiella pneumoniae isolates were extracted using a Quick-DNA™ miniprep kit (Zymo Research) per the manufacturer’s instructions. First, a loopful bacterial culture was taken from the Muller-Hinton agar plate and suspended in 1 ml nuclease-free water, and then 100 µl of it was taken into a 1.5 ml microcentrifuge tube. Next, 400 µl of was added to the microcentrifuge tube, mixed by vortexing for 5 seconds, and left to stand for 10 minutes at room temperature. Next, the mixture was transferred into a spin column in a collection tube and centrifuged at 10,000 x g for one minute. Next, the column was transferred into a new collection tube then 200 µl of DNA pre-wash buffer was added and centrifuged at 10,000 x g for one minute. Next, the column was transferred to a new collection tube then 500 µl of gDNA wash buffer was added, followed by centrifugation at 10,000 x g for one minute. Finally, the spin column was transferred to a clean microcentrifuge tube, then 50 µl of DNA elution buffer was added, incubated for 5 minutes, and then centrifuged at 10,000 x g for 30 seconds to elute the DNA. Next, DNA concentrations were measured using the Qubit™ machine (Thermofisher scientific) using the Qubit™ dsDNA HS Assay kit (Invitrogen).
 
## Oxford nanopore library preparation and sequencing
Library preparation was performed on extracted genomic DNA from the selected isolates (10 CRKP isolates) using the Oxford nanopore ligation sequencing kit SQK-LSK109 (Oxford Nanopore) and barcoding kit EXP-NBD104 (Oxford Nanopore). A detailed protocol (checklist) can be found in appendix III. Genomic DNA Sequencing was done overnight using an Oxford nanopore Mk101B device per manufacturer's instruction (see appendix III).
Primary data acquisition and real-time base calling were carried out using the Guppy basecaller v3.0.6. The demultiplexing of barcodes and quality control of the reads were accomplished by Guppy basecaller. The reads in fastq format from all samples were kept in separate folders according to barcode numbers.
 
## Illumina library preparation and sequencing
All eight carbapenem-resistant *Klebsiella pneumoniae* isolates were sent to Kilimanjaro Clinical Research Institute (KCRI) for Illumina sequencing under the SeqAfrica project. The sequencing library was prepared following the Illumina DNA preparation protocol. 
All libraries were sequenced on a NextSeq™ 550 system using NextSeq 500/500v2 High Output Kits (Illumina, Cat. No.FC-4042004) with a run configuration of 2 × 150bp to generate sufficient genomic coverage for de novo assembly. Base calling and quality scoring were performed with onboard NextSeq Control Software 7 and Real-Time Analysis v27 software.

## Reads quality control checks and quality trimming
NanoPlot (70) was used to assess the quality of fastq files from nanopore reads, followed by adopter trimming using Porechop (https://github.com/rrwick/Porechop). Then fastq files with similar barcodes were concatenated to form a single file.
Illumina reads were assessed using the FASTQC tool (https://github.com/s-andrews/FastQC), then reads with less than 30 Q-score were trimmed off using the Trimmomatic tool.
 
## Genome assembly and annotation
De novo assembly of nanopore reads was done using Flye with one round of polishing using Medaka software (https://github.com/nanoporetech/medaka). The assemblies were then polished using short reads from Illumina sequencing using the Polypolish tool (73), which first aligns the short reads into the assembled genome and then corrects the error found in the assembly.
Illumina only reads were assembled using the KCRI CGE Bacterial Analysis Pipeline (BAP) (https://github.com/kcri-tz/kcri-cge-bap). The pipeline evolved from the original CGE BAP (74). BAP uses SKESA to assemble the reads, and the pipeline can conduct downstream analyses, but in this study, the assembled genome was taken to conduct downstream analyses using other available tools. In addition, we assessed the quality of the genome assemblies using QUAST (75).
Annotation of genomic features in the assembly was achieved by using the Prokka tool (76). The polished and Illumina-only assemblies were annotated separately, and the genomic feature files (.gff) were used for downstream analyses.
 
## Identification of virulence, antimicrobial resistance genes and Plasmids
Antimicrobial resistance genes were detected by running the polished assemblies in Amrfinder plus (REF) software using the NCBI databases; we picked genes whose coverage and identity was greater than or equal to 90, the output was then analyzed using R.
The identification of antimicrobial resistance genes within the plasmids was done by querying the plasmid contigs in the center of the genomic epidemiology web server (http://www.genomicepidemiology.org/services/) (80). We identified the plasmid carried in the isolates by using the web based tool Pathogen watch (ref), which uses a plasmidfinder database hosted by the center of genomic epidemiology (ref).
 
## Multilocus sequence typing (MLST) 
The assembled genomes were mapped against the seven housekeeping genes, gapA, infB, mdh, pgi, phoE, rpoB and tonB, that make up the *Klebsiella pneumoniae* MLST scheme (52,65) to determine Multi-Locus Sequence Typing (MLST). The sequence types (ST) for each isolate were determined using Kleborate (84).
Illumina-only assemblies were used to type the isolates as they were more accurate (Q-score greater than 30). The isolates with novel alleles were submitted to BIGSdb-Pasteur databases at http://bigsdb.pasteur.fr/” for curation. MLSTs were confirmed using web based tool Pathogen watch (ref)
 
## Variant calling
We used snippy (REF) for variant calling using *Klebsiella pneumoniae* reference () for *Klebsiella pneumoniae* isolates and *Klebsiella variicola* reference () for *Klebsiella variicola* isolates.

## Patient's demographic data
Patient information regarding the onset of symptoms, admission to the hospital, age of patients, and patient history and antimicrobial use during the infection were taken from the patient's hospital file and laboratory information system. All data will be collected using the data collection form (see annexe 2) and compiled using Microsoft excel.
 
## Ethical considerations
MUHAS IRB granted the study ethical clearance number MUHAS-REC-02-2022-965, and a waiver of informed consent was given due to its retrospective nature, and no personal identifiers were used in this study

# Results
## Bacteria Isolates
A total of 143 Klebsiella pneumoniae were isolated from patients with bloodstream infections between August 2021 and March 2022. Of which 14% (20/143) were resistant to carbapenems. Sixteen isolates were taken, and selection was based on the ascending zone of inhibition to meropenem and/or imipenem. The 16 were taken for confirmation with the Vitek MS at the NPHL. eight isolates were confirmed to be Klebsiella pneumoniae, six were Enterobacter cloacae complex, one was Serratia ficaria, and the last was Lelliottia amnigena.  All eight confirmed Klebsiella pneumoniae were subjected to antimicrobial susceptibility testing and whole genome sequencing.
 
## Patients’ characteristics 
Five *Klebsiella pneumoniae* isolates were from neonates admitted to the neonatal ward and neonatal ICU, i.e., MNH_07 (isolated on 12/12/2021), MNH_01 (isolated on 10/01/2022), MNH_06 (isolated on 22/01/2022), MNH_04 (isolated on 27/02/2022) and MNH_10 (isolated on 04/03/2022), One isolate (MNH_05) was obtained from a toddler admitted to pediatric ICU on August 25, 2021. One isolate came from an adult MNH_15 from ICU (isolated on 05/03/2022). The table with descriptions of all patients from which the isolates came is in appendix:

## Phenotypic antimicrobial susceptibilities
The isolates were highly resistant to ceftriaxone (75%), ceftazidime (87.5%), gentamicin (62.5%) and trimethoprim-sulfamethoxazole (50%). The least resistance was observed to amikacin (0%) and tetracycline (42.9%). Resistance to meropenem and imipenem were 87.5% and 0%, respectively, table 2. 

## Antimicrobial Resistance  and virulence genes


## Multi-locus Sequence Typing
In MLST, six out of eight samples were confirmed to be *Klebsiella pneumoniae*, and two were identified as *Klebsiella variicola subsp. Variicola*. All the six *Klebsiella pneumoniae* isolates had a unique sequence type (ST), MNH_04 belonged to ST1401, MNH_05 belonged to ST11, MNH_06 was ST2279, MNH_10 belonged to ST14, and MNH_16 was from ST2816 Isolate MNH_15 had a novel sequence type with novelty in three genes gapA, infB, and rpoB due to poor assemblies resulting from low depth and coverage during sequencing. For the two *Klebsiella variicola subsp. variicola* isolates belonged to ST6257 which is also a novel ST.


# Discussion

# Conclusion

# Acknowledgement


# Authors contribution

# Funding 











# Bibliography styles

Here are two sample references: @Feynman1963118 [@Dirac1953888].

By default, natbib will be used with the `authoryear` style, set in `classoption` variable in YAML and with `elsearticle-harv.bst` which is among provided style by `elsarticle` documentclass. Other available style are `elsarticle-num.bst` and `elsarticle-num-names.bst` — the first one can be used for the numbered scheme, second one for numbered with new options of natbib.sty. 

You can sets extra options with `natbiboptions` variable in YAML header. Example 
```yaml
natbiboptions: longnamesfirst,angle,semicolon
```

There are various more specific bibliography styles available at
<https://support.stmdocs.in/wiki/index.php?title=Model-wise_bibliographic_style_files>. 
To use one of these, add it in the header using, for example, `biblio-style: model1-num-names`.

## Using CSL 

If `citation_package` is set to `default` in `elsevier_article()`, then pandoc is used for citations instead of `natbib`. In this case, the `csl` option is used to format the references. Alternative `csl` files are available from <https://www.zotero.org/styles?q=elsevier>. These can be downloaded
and stored locally, or the url can be used as in the example header.

# Equations

Here is an equation:
$$ 
  f_{X}(x) = \left(\frac{\alpha}{\beta}\right)
  \left(\frac{x}{\beta}\right)^{\alpha-1}
  e^{-\left(\frac{x}{\beta}\right)^{\alpha}}; 
  \alpha,\beta,x > 0 .
$$

Here is another:
\begin{align}
  a^2+b^2=c^2.
\end{align}

Inline equations: $\sum_{i = 2}^\infty\{\alpha_i^\beta\}$

# Figures and tables

Figure \ref{fig2} is generated using an R chunk.

```{r fig2, fig.width = 5, fig.height = 5, fig.align='center', out.width="50%", fig.cap = "\\label{fig2}A meaningless scatterplot.", echo = FALSE}
plot(runif(25), runif(25))
```

# Tables coming from R

Tables can also be generated using R chunks, as shown in Table \ref{tab1} for example.

```{r tab1, echo = TRUE}
knitr::kable(head(mtcars)[,1:4], 
    caption = "\\label{tab1}Caption centered above table"
)
```

# References {-}

